src/util.cc

```diff
@@ -725,13 +727,11 @@ void SetConstructorFunction(Isolate* isolate,
   that->Set(name, tmpl);
 }
 
-Local<String> UnionBytes::ToStringChecked(Isolate* isolate) const {
+MaybeLocal<String> UnionBytes::ToString(Isolate* isolate) const {
   if (is_one_byte()) {
-    return String::NewExternalOneByte(isolate, one_byte_resource_)
-        .ToLocalChecked();
+    return String::NewExternalOneByte(isolate, one_byte_resource_);
   } else {
-    return String::NewExternalTwoByte(isolate, two_byte_resource_)
-        .ToLocalChecked();
+    return String::NewExternalTwoByte(isolate, two_byte_resource_);
```

Comment:
This change kind of only brings us halfway towards the right semantics, because the comment in this snippet also applies here: https://github.com/nodejs/node/blob/15278252bb8391ef78f6973d15e4834da7030de9/src/util-inl.h#L347-L357